build.sbt

```diff
@@ -37,6 +37,7 @@ libraryDependencies ++= Seq(
   "org.apache.spark" %% "spark-sql-kafka-0-10" % sparkVersion % "provided" excludeAll(excludeJpountz),
   "org.apache.spark" %% "spark-streaming" % sparkVersion % "provided",
   "org.apache.spark" %% "spark-avro" % sparkVersion % "provided",
+  "com.springml" % "spark-zuora_2.11" % "1.1.0",
```

Comment:
Wondering if this should be provided...

Reply:
@lyogev if it has different versions for different spark versions than yes